core/src/main/java/com/backblaze/b2/client/structures/B2ApplicationKey.java

```diff
@@ -45,15 +49,17 @@
                     "capabilities, " +
                     "bucketId, " +
                     "namePrefix, " +
-                    "expirationTimestamp"
+                    "expirationTimestamp, " +
+                    "options"
     )
     public B2ApplicationKey(String accountId,
```

Comment:
Incompatible change?  Does this need a major version number bump?

Reply:
updated

Reply:
the B2Json.constructor annotation can only be used once per class.

Looks like there isn't a unit test for this class. Fabian, can you please create one for it? Running that test should result in an error due to having two constructor annotations

Reply:
i'm fine with bumping the version.  in fact, fabian & i just looked and we've bumped to v4 because of other little (breaking) changes and haven't released yet, so bumping is free.  :)

Reply:
re: unit test malay suggested -- one particularly good test is to make a structure, toJson it, fromJson it, and verify that the original and the copy match with equals.  you can also toJson again to be sure that the two json versions also match, but i'm not sure that's strictly necessary most of the time (at least not if you trust your equals method).

Reply:
thanks for suggestion - updated